pysmt/cmd/installers/cvc4.py

```diff
@@ -61,9 +61,10 @@ def compile(self):
         PYTHON_LIBRARY = os.environ.get('PYSMT_PYTHON_LIBDIR')
         if not PYTHON_LIBRARY:
             PYTHON_LIBRARY = sysconfig.get_config_var('LIBDIR')
-        PYTHON_INCLUDE_DIR = sysconfig.get_python_inc()
-        SolverInstaller.run(['sed', '-i',
-                             's|cmake_opts=""|cmake_opts="-DPYTHON_LIBRARY=' + PYTHON_LIBRARY + ' -DPYTHON_INCLUDE_DIR=' + PYTHON_INCLUDE_DIR + '"|g',
+        PYTHON_LIBRARY = self.__add_backslash(PYTHON_LIBRARY)
+        PYTHON_INCLUDE_DIR = self.__add_backslash(sysconfig.get_python_inc())
+        SolverInstaller.run(["perl", "-i''", "-pe",
```

Comment:
why using `perl` instead of `sed`? The latter is much more common (at least in the linux world).
If on OSX this is not available, you can condition the use of `perl` by checking the platform:
```python
if self.os_name == 'darwin':
    ...
```